### DISCUSSION TITLE
--notebook replacement?

### DISCUSSION THREAD

**Discussion by ghost** [oobabooga/text-generation-webui#4106] [Ideas]
What to use now that the command is deprecated? Trying to set it in settings.yaml doesn't work.

**Comment by FartyPants** [comment]
because notebooks are on main GUI regardless, look on the top menu.

**Reply by ghost** [reply]
That's very cool but I don't see the point of deleting a QoL feature that just worked for no reason. Just why?

**Reply by FartyPants** [reply]
You have to ask the person who deleted it.

**Reply by ghost** [reply]
That's what I hoped would happen by posting here.

**Comment by ghost** [comment]
Why is this marked as resolved?
